training-net/src/Controllers/MoviesController.cs

```diff
@@ -17,12 +21,10 @@ public MoviesController(IHtmlLocalizer<MoviesController> localizer, IUnitOfWork
             this._localizer = localizer;
             this._unitOfWork = unitOfWork;
         }
-        
-        public IActionResult Index()
-        {   
-            ViewData["Message"] = Localizer["DescriptionPage"];
-            return View();
-        }
+
+        public IActionResult Index() =>
+        View(UnitOfWork.Movies.GetAll().ToList().ConvertAll(x => { return new MovieViewModel{ID=x.ID,Title=x.Title,ReleaseDate=x.ReleaseDate,Genre=x.Genre,Price=x.Price};}));
```

Comment:
it should be corrected in the last pr, but just in case, remember to replace the convert all and the spaces between properteis